src/index.html

```diff
@@ -1,13 +1,58 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Stars title</title>
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="page">
+      <section class="stars page__stars">
+        <ul class="list stars__list stars--0">
```

Comment:
`<ul>` elements could be separated by an empty line.

Reply:
> `<ul>` elements could be separated by an empty line.

Fixed